Formula/clipboard.rb

```diff
@@ -0,0 +1,40 @@
+class Clipboard < Formula
+  desc "Cut, copy, and paste anything, anywhere, all from the terminal"
+  homepage "https://getclipboard.app"
+  url "https://github.com/Slackadays/Clipboard/archive/refs/tags/0.3.1.tar.gz"
+  sha256 "c00580e60c7c402bb0b8e87133fdc511b25205200adb2c17e96c5811cc79c37d"
+  license "GPL-3.0-or-later"
+  head "https://github.com/Slackadays/Clipboard.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on macos: :monterey
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "wayland-protocols" => :build
+    depends_on "libx11"
+    depends_on "wayland"
+  end
+
+  fails_with :clang do
+    build 1300
+    cause "Requires C++20 support"
+  end
+
+  def install
+    args = std_cmake_args
+    args << "-DINSTALL_PREFIX=#{prefix}"
+    system "cmake", "-S", ".", "-B", "build", *args
```

Comment:
The prefix is set by the standard arguments
```suggestion
    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
```

Reply:
args set `CMAKE_INSTALL_PREFIX`, this package requires a custom INSTALL_PREFIX instead

Reply:
Could that be fixed upstream ?

Reply:
A question to @Slackadays, but afaik it was explicitly switched in recent release.

Reply:
Done